libconcurrent/run_all.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Remove the resuts file if exists
+if [ -f results.txt ] ; then
```

Comment:
I think, it is a goo practice to set a variable RESULTS="results.txt",
just in case a use may easily want to change the output file.
Also this may catch some easy bugs, such that of line 30.

Reply:
Will do

Reply:
Done